ui/components/app/metamask-template-renderer/metamask-template-renderer.stories.js

```diff
@@ -85,25 +91,12 @@ const SECTIONS = {
   ],
 };
 
-export const DefaultStory = () => (
-  <MetaMaskTemplateRenderer sections={object('sections', SECTIONS)} />
+export const DefaultStory = (args) => (
+  <MetaMaskTemplateRenderer sections={args.sections} />
 );
 
 DefaultStory.storyName = 'Default';
 
-export const WithInvalidElement = () => (
-  <MetaMaskTemplateRenderer
-    sections={object('sections', [
-      {
-        ...SECTIONS,
-        key: 'safe-tree',
-      },
-      {
-        element: 'Unsafe',
-        key: 'unsafe-tree',
-        children:
-          'I should be displayed, but I wont be due to unsafe component',
-      },
-    ])}
-  />
-);
```

Comment:
I suggested to @dj-pundir to remove this story as it was display an error intentionally but I don't think it adds much value in storybook